packages/storybook/src/nimble/combobox/combobox.stories.ts

```diff
@@ -113,14 +115,20 @@ const metadata: Meta<ComboboxArgs> = {
             appearance="${x => x.appearance}"
             value="${x => x.currentValue}"
             placeholder="${x => x.placeholder}"
-            style="width: 250px;"
+            style="min-width: 250px;"
```

Comment:
Note: Switching from `width` to `min-width` in these stories so that the control will grow with a long label, instead of wrapping the label (matches more of our other control stories).

Reply:
The theme matrix stories have wrapping labels because they set a fixed `width` on the control. This matches the other stories for controls with labels (and I think a fixed width makes sense for the theme matrix so everything stays in a grid).

Reply:
Think it's expected in matrix tests so the controls layout nicely. But for the doc stories it's allowed to grow